cmake/CMakeLists.txt

```diff
@@ -187,12 +181,10 @@ if(ENABLE_MSCG OR ENABLE_USER-ATC OR ENABLE_USER-AWPMD OR ENABLE_USER-QUIP)
     if(BUILD_SHARED_PKG_LIBS)
       add_library(linalg SHARED ${LAPACK_SOURCES})
       set_target_properties(${PKG_LIB} PROPERTIES SOVERSION ${SOVERSION})
-      if(INSTALL_LIB)
-        install(TARGETS linalg LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-      endif()
     else()
       add_library(linalg STATIC ${LAPACK_SOURCES})
     endif()
+    install(TARGETS linalg LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
same here, you moved the install from the shared branch to both cases. this isn't what I had in mind. seems like I should have been more explicit...